packages/opentelemetry/src/sampler.ts

```diff
@@ -68,7 +84,10 @@ export class SentrySampler implements Sampler {
     // The reason for this is that the data quality of the spans varies, it is different per version of Next,
     // and we need to keep our manual instrumentation around for the edge runtime anyhow.
     // BUT we only do this if we don't have a parent span with a sampling decision yet (or if the parent is remote)
-    if (spanAttributes['next.span_type'] && (typeof parentSampled !== 'boolean' || parentContext?.isRemote)) {
+    if (
+      (spanAttributes['next.span_type'] || NEXTJS_SPAN_NAME_PREFIXES.includes(spanName.split('.')[0])) &&
```

Comment:
hmm this souns a bit like it could lead to false positives 😬 is there no other attribute type we can check for (e.g. `next.xxx`)? 🤔

Reply:
I think that's the cost we have to pay if we have this so central in the opentelemetry package. Imo would it be isolated to the nextjs sdk it's probably fine but youre right that here its probably too risky.